check-cla/action.yml

```diff
@@ -3,161 +3,186 @@ name: CLA check
 description: Reacts to new PRs and check if the contributor has previously signed the conda contributor license agreement (CLA).
 inputs:
   token:
-    description: Token for commenting and labeling on contributor's PR
-    required: true
-  infrastructure_token:
-    description: Token for opening singee PR in conda/infrastructure
+    description: 'A token with ability to comment, label, and modify the commit status (`pull_request: write` and `statuses: write`)'
+    default: ${{ github.token }}
     required: true
   label:
     description: Label to apply to contributor's PR once CLA is singed
     required: true
+  cla_repo:
+    description: Upstream repository in which to create PR
+    default: conda/infrastructure
+  cla_path:
+    description: Path to the CLA signees file within the provided `cla_repo`
+    default: .clabot
+  cla_fork:
+    description: Fork of `cla_repo` in which to create branch
+    default: conda-bot/infrastructure
+  cla_token:
+    description: Token for opening singee PR in `cla_fork`
+    required: true
+  cla_author:
+    description: Git-format author/committer to use for pull request commits
+    default: Conda Bot <18747875+conda-bot@users.noreply.github.com>
 
 runs:
   using: composite
   steps:
-    - name: Get PR metadata
-      id: pr
+    # if triggered by a comment, leave a reaction
+    - name: Comment Reaction
+      uses: peter-evans/create-or-update-comment@v3.0.0
+      if: github.event_name == 'issue_comment'
+      with:
+        token: ${{ inputs.token }}
+        comment-id: ${{ github.event.comment.id }}
+        reactions: eyes
+
+    # has_label
+    - name: Get PR labels
+      id: labels
       uses: actions/github-script@v6
       with:
         script: |
           const { owner, repo, number } = context.issue;
-          const pullRequest = await github.rest.pulls.get({
-            owner,
-            repo,
-            pull_number: number,
+          core.debug(`owner: ${owner}`);
+          core.debug(`repo: ${repo}`);
+          core.debug(`number: ${number}`);
+
+          const raw = await github.rest.pulls.get({
+            owner: owner,
+            repo: repo,
+            pull_number: number
           });
-          console.log(pullRequest);
-          const sha = pullRequest.data.head.sha;
-          console.log(sha);
-          core.setOutput('sha', sha);
-
-          const labels = pullRequest.data.labels.map(label => label.name)
-          console.log(labels);
-          core.setOutput('labels', labels);
+          const labels = raw.data.labels.map(label => label.name)
+          core.debug(`labels: ${labels}`);
 
-          const hasLabel = labels.includes('${{ inputs.label }}')
-          console.log(hasLabel);
-          core.setOutput('hasLabel', hasLabel);
+          const has_label = labels.includes('${{ inputs.label }}')
+          core.setOutput('has_label', has_label);
+          core.debug(`has_label: ${has_label}`);
```

Comment:
Moved `sha` detection into new `set-commit-status` action. Removed `labels` output since we don't use it outside of this scope. Switch `hasLabel` to `has_label` for variable consistency.

Reply:
See https://github.com/conda/actions/pull/91/files#r1158994208